zebra-chain/src/parameters/network_upgrade.rs

```diff
@@ -40,8 +40,8 @@ pub enum NetworkUpgrade {
     /// The Zcash protocol after the NU5 upgrade.
```

Comment:
Maybe use `fastmod` or similar to catch them all, i think we should changes the ones that are in the text as well for consistency.

Reply:
Sorry I already merged this PR, happy to take a follow-up PR that fixes the missing ones. (Not a high priority though.)

I'd also be happy to have the comments and strings say `NU5`, but the code say `Nu5`.